server/internal/heavy/components.go

```diff
@@ -252,14 +252,14 @@ func newComponents(ctx context.Context, cfg configuration.Configuration, genesis
 		Handler      *handler.Handler
 		Genesis      *genesis.Genesis
 		Records      *object.RecordDB
-		JetKeeper    executor.JetKeeper
+		JetKeeper    *executor.DBJetKeeper
```

Comment:
Why not interface?

Reply:
Because I need to forward to to different interfaces